x11/mesa/Portfile

```diff
@@ -36,24 +36,30 @@ meson.wrap_mode         nodownload
 set py_ver              3.12
 set py_ver_nodot        [string map {. {}} ${py_ver}]
 
+set llvm_ver            19
+
 depends_build-append    port:pkgconfig \
                         port:flex \
                         port:bison \
-                        port:py${py_ver_nodot}-mako \
                         port:python${py_ver_nodot} \
+                        port:py${py_ver_nodot}-mako \
+                        port:py${py_ver_nodot}-yaml \
                         port:xorg-xorgproto
 
 depends_lib-append      port:expat \
                         port:xorg-libxcb \
                         port:xorg-libX11 \
                         port:xorg-libXext \
                         port:xorg-libXdamage \
+                        port:xorg-libXrandr \
+                        port:llvm-${llvm_ver} \
```

Comment:
This will break every OS where llvm19 is absent. It should obviously be conditional. If LLVM is really required and cannot be avoided, then older OS need a peg to LLVM-free `mesa`.

Reply:
It is not needed at all in fact, it is an optional feature. See `meson_options`.

Reply:
There is some more nonsense in their sources to fix, let me sort that out.